drizzle-orm/src/pg-core/query-builders/delete.ts

```diff
@@ -141,7 +142,7 @@ export class PgDeleteBase<
 	returning(
 		fields: SelectedFieldsFlat = this.config.table[Table.Symbol.Columns],
 	): PgDeleteReturning<this, TDynamic, any> {
-		this.config.returning = orderSelectedFields(fields);
+		this.config.returning = orderSelectedFields<PgColumn>(fields);
```

Comment:
This might be left over code from your initial implementation. Or is it necessary to make the compiler happy?

Reply:
A bit of both. Yes, it's leftover from the previous implementation, but decided to leave it as it has nothing to do with the `BuiltInFunction` class. This makes it consistent with the MySQL dialect, as `MySqlColumn` was specified there but not in PG or SQLite for some reason.

Reply:
@Angelelz Let me know if you think this should stay or not.